app/controllers/make_a_demo_payment/index_controller.js

```diff
@@ -33,6 +33,8 @@ module.exports = (req, res) => {
   const currencyMatch = AMOUNT_FORMAT.exec(paymentAmount)
```

Comment:
Not a deal breaker, but maybe we could extract this bit in a function, with a name which makes it possible to understand what's it doing, at a glance? 
Just reading the code it's quite hard to understand what's going on - it's padding the amount, but what's the logic behind it? :)